### ISSUE TITLE
topic representation

### ISSUE THREAD

**Issue by nuonuo1013** [MaartenGr/BERTopic#1332]
Thanks for the code, but I have a question：Is it possible to show more than 10 representative words in each topic?

**Comment by MaartenGr** [comment]
Yes, you can use the parameter `top_n_words` for that. Also, make sure to check that parameter in the representation models if you are using it.

**Comment by nuonuo1013** [comment]
Thank you very much for your answer, one more question is whether `print (topic_model.topics_over_time)` can output more words to observe topic timing changes when modeling dynamically
